WooCommerce/Resources/Assets.xcassets/LaunchLogo.imageset/Contents.json

```diff
@@ -1,15 +1,15 @@
 {
   "images" : [
     {
-      "idiom" : "universal",
-      "filename" : "woo-icon-120x120.pdf"
```

Comment:
The 120x120 part of the name was misleading here, as the file was actually 164x90.

I decided to not put the resolution there now, I think it doesn't bring any real value, XCode shows the resolution in the Assets manager.